core/chaincode/chaincode_support.go

```diff
@@ -122,6 +122,13 @@ func NewChaincodeSupport(chainname ChainName, getPeerEndpoint func() (*pb.PeerEn
 		s.chaincodeInstallPath = chaincodeInstallPathDefault
 	}
 
+	s.peerTLS = viper.GetBool("peer.tls.enabled")
+	if s.peerTLS {
+		s.peerTLSCertFile = viper.GetString("peer.tls.cert.file")
+		s.peerTLSKeyFile = viper.GetString("peer.tls.key.file")
```

Comment:
I don't think there is any need to add the `peer.tls.key.file` since this is a client connection and we only need the cert so that it can be trusted in the TLS client connection.